apps/login-register/src/app/login/login.component.ts

```diff
@@ -10,6 +12,22 @@ import { Router } from '@angular/router';
 export class LoginComponent implements OnInit {
   loginForm: FormGroup = new FormGroup({});
 
+  errorMessages: any = {
+    emailMessages: '',
```

Comment:
extract this into a separate file constants.ts

Reply:
same